web-admin/src/features/embeds/TopNavigationBarEmbed.svelte

```diff
@@ -70,13 +74,22 @@
         {/if}
 
         {#if currentResource}
-          <BreadcrumbItem
-            options={breadcrumbOptions}
-            current={currentResourceName}
-            onSelect={onSelectResource}
-            isCurrentPage
-            isEmbedded
-          />
+          {#if !twoTieredNavigation}
```

Comment:
Nit: it'd be more clear if the conditional did not check the negative, so `{#if twoTieredNavigation} ...`